src/PKSim.Core/Services/IndividualExpressionsUpdater.cs

```diff
@@ -24,9 +27,20 @@ public void Update(Individual sourceIndividual, Individual targetIndividual)
          {
             var newMolecule = _cloner.Clone(molecule);
             targetIndividual.AddMolecule(newMolecule);
-            //we have to reset the new ontogeny for the molecule
+
+            //Make sure parameters that user defined parameters are reset to default to ensure proper scaling
+            resetMoleculeParametersToDefault(newMolecule);
```

Comment:
At the moment, molecule were cloned and added. When the user changed the default value, the "default value " of the target value was wrong. Parameter needs to be reset to default to see the scaling magic happen